package.json

```diff
@@ -31,15 +31,14 @@
     "broccoli-asset-rev": "^2.4.2",
     "ember-browserify": "1.1.13",
     "ember-cli": "~2.8.0",
-    "ember-cli-app-version": "^1.0.0",
     "ember-cli-code-coverage": "~0.3.0",
     "ember-cli-content-security-policy": "^0.5.0",
     "ember-cli-dependency-checker": "^1.2.0",
     "ember-cli-ecosystem-installer": "2.0.25",
     "ember-cli-ember-ecosystem": "0.4.0",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-mocha": "^0.10.0",
+    "ember-cli-mocha": "~0.13.0",
```

Comment:
Cause of your build failure - chai is now provided by the `ember-cli-chai` addon which will be added to your projects by default during ember install ember-cli-mocha.
If you upgrade your version of ember-cli-mocha you will have to run ember generate ember-cli-mocha to make sure the dependency gets installed in your project!